sorc/gfs_bufr.fd/meteorg.f

```diff
@@ -1065,7 +1065,7 @@ subroutine meteorg(npoint,rlat,rlon,istat,cstat,elevstn,
           psn(np) = ps(np)
           call sigio_modpr(1,1,levs,nvcoord,idvc,
      &         idsl,vcoord,iret,
-     &         ps=psn(np)*1000,pd=pd3(np,1:levs))
+     &         ps=psn(1:np)*1000,pd=pd3(1:np,1:levs))
```

Comment:
What is the reason for this change?

Reply:
These minor tweaks I make are needed for GNU builds. If I don't do that I get

```
meteorg.f:1068:18:

      &         ps=psn(np)*1000,pd=pd3(np,1:levs))
                  1
Error: Rank mismatch in argument 'ps' at (1) (rank-1 and scalar)
meteorg.f:1106:50:
```